pkg/config/configparameters.go

```diff
@@ -13,35 +13,37 @@ import (
 )
 
 type ConfigParameters struct {
-	FoodOnPlatform food.FoodType `json:"FoodOnPlatform"`
-	Team1AgtOne    int           `json:"Team1AgtOne"`
-	Team1AgtTwo    int           `json:"Team2AgtTwo"`
-	Team2Agents    int           `json:"Team2Agents"`
-	Team3Agents    int           `json:"Team3Agents"`
-	Team4Agents    int           `json:"Team4Agents"`
-	Team5Agents    int           `json:"Team5Agents"`
-	Team6Agents    int           `json:"Team6Agents"`
-	Team7Agent1    int           `json:"Team7Agent1"`
-	RandomAgents   int           `json:"RandomAgents"`
-	AgentHP        int           `json:"AgentHP"`
-	AgentsPerFloor int           `json:"AgentsPerFloor"`
-	TicksPerFloor  int           `json:"TicksPerFloor"`
-	SimDays        int           `json:"SimDays"`
-	ReshuffleDays  int           `json:"ReshuffleDays"`
-	MaxHP          int           `json:"maxHP"`
-	WeakLevel      int           `json:"weakLevel"`
-	Width          float64       `json:"width"`
-	Tau            float64       `json:"tau"`
-	HpReqCToW      int           `json:"hpReqCToW"`
-	HpCritical     int           `json:"hpCritical"`
-	MaxDayCritical int           `json:"maxDayCritical"`
-	HPLossBase     int           `json:"HPLossBase"`
-	HPLossSlope    float64       `json:"HPLossSlope"`
-	LogFileName    string        `json:"LogFileName"`
-	NumOfAgents    []int
-	NumberOfFloors int
-	TicksPerDay    int
-	DayInfo        *day.DayInfo
+	FoodOnPlatform       food.FoodType `json:"FoodOnPlatform"`
+	FoodPerAgentRatio    int           `json:"FoodPerAgentRatio"`
+	UseFoodPerAgentRatio bool          `json:"UseFoodPerAgentRatio"`
+	Team1AgtOne          int           `json:"Team1AgtOne"`
+	Team1AgtTwo          int           `json:"Team2AgtTwo"`
```

Comment:
shouldn't that be `json:"Team1AgtTwo"`

Reply:
From a quick look this seems to be left from Team2's commit from PR #148. I saw that you also fixed it in your latest commit, and I incorporated those changes in my rebase.